published_topic_names:
  sound_source_localisation: /sound/alert
subscribed_topic_names:
  audio_stream: /pandora_audio/kinect_audio_capture_stream